release/models/interfaces/openconfig-if-aggregate.yang

```diff
@@ -148,7 +148,7 @@ module openconfig-if-aggregate {
       units Mbps;
       description
         "Reports effective speed of the aggregate interface,
-        based on speed of active member interfaces";
+        based on speed of active (operationally up) and forwarding-viable member interfaces";
```

Comment:
In discussion at OC Operators meeting, it was mentioned that LACP backup mode should also affect the lag-speed.  (ie: a member in 'backup' state should not be included in lag-speed).   I think it's helpful to enumerate the conditions we know of here in the description to help increase clarity for everyone.